src/main/java/org/hmhb/program/Program.java

```diff
@@ -0,0 +1,159 @@
+package org.hmhb.program;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.hmhb.organization.Organization;
+
+@Entity
+public class Program {
```

Comment:
Published flag doesn't allow multiple (you request updating an existing, or multiple people put in different requests).  I think it will be another table one way or another (whether I model it as a OneToMany property of program or not, I'm not sure).

As for the address to geo stuff, I'm not sure yet.  I need to dig around into google's api before I'll know enough to take a stab at the workflow + data modeling of it.